src/menu/menu.c

```diff
@@ -293,9 +306,16 @@ separator_create(struct menu *menu, const char *label)
 	struct server *server = menu->server;
 	struct theme *theme = server->theme;
 
+	/*
+	 * Convert empty label ie "", to a regular separator line
+	 * if one desires an empty title line then set label to " "
+	 */
+	if (label && !strlen(label)) {
```

Comment:
Delete. This is the same as three lines up... `menuitem->type = string_null_or_empty(label) ? LAB_MENU_SEPARATOR_LINE : LAB_MENU_TITLE;`